src/App.tsx

```diff
@@ -2,17 +2,25 @@ import { PeoplePage } from './components/PeoplePage';
 import { Navbar } from './components/Navbar';
 
 import './App.scss';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { NotFoundPage } from './components/NotFoundPage';
+import { HomePage } from './components/HomePage';
 
 export const App = () => {
   return (
     <div data-cy="app">
       <Navbar />
-
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/" element={<HomePage />} />
+            <Route path="home" element={<Navigate to="/" replace />} />
+            <Route path="/people">
+              <Route index element={<PeoplePage />} />
+              <Route path=":slug?" element={<PeoplePage />} />
+            </Route>
```

Comment:
The nested route for `PeoplePage` is defined correctly, but the optional parameter `:slug?` might not be necessary if the `index` route already serves the purpose. Ensure that both routes are needed, or consider removing the optional parameter if it's redundant.